cpp/ps0008/problem. 8.cpp

```diff
@@ -38,11 +44,16 @@ int main() {
         } else {
             std::cout << "Wrong input, try again" << std::endl;
         }
-        }
+    }
+    
     std::cout << "Enter the encrypted word or sentence:\n  ";
-    std::cin.ignore(); 
-    std::getline(std::cin, input);
+    std::cin.ignore();
+    std::getline(std::cin, input); // Taking message 1/1
+    
+    // invoke your decrypt function to return the plaintext 1/1
     std::string decryptedText = decrypt(input, key);
     std::cout << "Decrypted text: " << decryptedText << std::endl;
```

Comment:
- [x] invoke a function for decryption process
- [x] printed the plaintext (actual message before encryption)